draft-ietf-quic-transport.md

```diff
@@ -1453,13 +1453,24 @@ ID values as the client's first Initial.
 
 Upon first receiving an Initial or Retry packet from the server, the client uses
 the Source Connection ID supplied by the server as the Destination Connection ID
-for subsequent packets, including any subsequent 0-RTT packets.  That means that
-a client might change the Destination Connection ID twice during connection
-establishment, once in response to a Retry and once in response to the first
-Initial packet from the server. Once a client has received an Initial packet
+for subsequent packets, including any 0-RTT packets.  A client changes the
+Destination Connection ID field if it receives a Retry packet.  A client always
+changes the Destination Connection ID field in response to the first Initial
+packet received from a server.  Once a client has received an Initial packet
 from the server, it MUST discard any packet it receives with a different Source
 Connection ID.
 
+Each endpoint includes the Source Connection ID it includes in Initial packets
```

Comment:
```suggestion
Each endpoint includes its Initial packet Source Connection ID 
```

Reply:
Thanks.  I've tried to avoid that sort of train of nouns in the past.  I have an alternative for you.  More words again :(, but maybe clearer.

Reply:
No worries! Just a suggestion in reaction to redundant "includes."